src/components/PopupWithForm.js

```diff
@@ -3,35 +3,29 @@ import Popup from "./Popup.js";
 export default class PopupWithForm extends Popup {
   constructor(popupSelector, handleFormSubmit) {
     super(popupSelector);
+    this._submitButton = this._popupElement.querySelector(".modal__button");
     this._handleFormSubmit = handleFormSubmit;
     this._popupForm = this._popupElement.querySelector(".modal__form");
-    console.log(this._popupForm);
     this._inputList = this._popupForm.querySelectorAll(".modal__input");
-    console.log(this._inputList);
   }
 
   _getInputValues() {
     const formValues = {};
     this._inputList.forEach((input) => {
       formValues[input.name] = input.value;
     });
-    console.log(formValues);
     return formValues;
   }
 
```

Comment:
**Could be improved**

You can make ` setInputValues` in class `PopupWithForm`, which can insert data into inputs:

```js

  setInputValues(data) {
    this._inputList.forEach((input) => {
      // Here you insert the `value` by the `name` of the input
      input.value = data[input.name];
    });
  }



```

So, this way you’ll not have to search the inputs of the profile in `index.js`